frontend/public/components/console-notifier.jsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import * as _ from 'lodash-es';
+
+import { consoleNotificationsStateToProps } from '../extensions/extensions-reducers';
+
+export const ConsoleNotifier = connect(consoleNotificationsStateToProps)(({notifications}) => !_.isEmpty(notifications)
```

Comment:
If we want watch, we could use `Firehose` in `ConsoleNotifier` as skip the other discovery on startup